article/helpers.py

```diff
@@ -56,3 +72,15 @@ def retrieve_articles(self):
         )
         response.raise_for_status()
         return [article['article_uuid'] for article in response.json()]
+
+    def article_read_count(self, category):
+        if category not in structure.CATEGORIES_LOOKUP:
+            raise Exception('Incorrect category')
+
+        read_articles = frozenset(self.retrieve_articles())
+        articles_in_category = frozenset(
```

Comment:
Articles in `ArticleGroup` need to be a list because they have to be ordered. Turning a list into a frozenset is a very cheap operation, there are not very many articles (50?) and not very many users. Explicitly casting the list to a frozenset here, where the set is the intersected, makes the code more readable.